calicoctl/calico_ctl/container.py

```diff
@@ -506,7 +506,8 @@ def get_ip_and_pool(ip):
             ip = result[1][0]
         pool = get_pool_or_exit(ip)
     elif ip is not None and '/' in ip:
-        pool = IPPool(ip)
+        cidr = IPNetwork(ip)
+        pool = get_pool_or_exit(IPAddress(cidr.first))
```

Comment:
It's only because `client.get_pool` accepts only `IPAddress` object, so just use the first IP of the CIDR will do the same effect only if there's no intersection for any of each two CIDRs. If they do, won't anything wrong happen when use `client.get_pool`... ? Anyway I will change this as you said...